manuscript/main.tex

```diff
@@ -1046,8 +1060,21 @@ \subsubsection*{Machine learning for cognitive development}
 representations of the gut microbiome (taxonomic profiles, functional
 profiles encoded as ECs) would behave, with and without the addition of
 demographics (sex and maternal education as a proxy of socioeconomic
-status) on participants from different age groups. Age (in months) was
-provided as a covariate for all models (Table S3).
```

Comment:
This is no longer true?

Reply:
It is. The sentence was moved to the end of the pargraph, line ~1077